web/src/copy_and_paste.ts

```diff
@@ -400,17 +417,14 @@ export function paste_handler_converter(paste_html: string): string {
     assert(copied_html_fragment !== null);
     const copied_within_single_element = within_single_element(copied_html_fragment);
     const outer_elements_to_retain = ["PRE", "UL", "OL", "A", "CODE"];
-    // If the entire selection copied is within a single HTML element (like an
-    // `h1`), we don't want to retain its styling, except when it is needed to
-    // identify the intended structure of the copied content.
+    const firstChild = copied_html_fragment.firstElementChild;
     if (
         copied_within_single_element &&
-        copied_html_fragment.firstElementChild !== null &&
-        !outer_elements_to_retain.includes(copied_html_fragment.firstElementChild.nodeName)
+        firstChild !== null &&
+        !outer_elements_to_retain.includes(firstChild.nodeName)
     ) {
-        paste_html = copied_html_fragment.firstElementChild.innerHTML;
+        paste_html = firstChild.innerHTML;
```

Comment:
Is this block just extracting a variable `firstChild`? Please do that sort of refactor in a prep commit so it is easy to review. I'm also not sure why you deleted the comment?

Check out our [GitHub guide](https://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details.

Reply:
This is just a refactor to make the code cleaner by avoiding repetitive calls to copied_html_fragment.firstElementChild within the condition.